package.json

```diff
@@ -61,5 +61,8 @@
   },
   "peerDependencies": {
     "three": ">=0.123.0"
+  },
+  "dependencies": {
+    "@react-three/fiber": "^8.17.9"
```

Comment:
This should be added under [optionalDependencies](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#optionaldependencies) in addition to `react` and `react-dom` since those are used in the project.